decidim-core/app/helpers/decidim/tooltip_helper.rb

```diff
@@ -4,9 +4,16 @@ module Decidim
   # This helper includes some methods to help with the inclusion of tooltips
   # on the layout.
   module TooltipHelper
+    # redesign deprecated
     def with_tooltip(title, &)
       content_tag(:span, data: { tooltip: true, disable_hover: false, keep_on_hover: true, click_open: false },
                          title:, &)
     end
+
+    def decidim_tooltip(opts = {}, &)
```

Comment:
I think the naming and structure of the old method above (the deprecated) was better for this purpose. I.e. the content within will be bound to the provided tooltip, it says it better. Looking at the implementation, would make more sense to me at least.

It would be also good for compatibility reasons to use the same approach.

Looking at the core, I see that the `with_tooltip` method is not so widely used, so maybe it would work simply replacing that method? Did you try that?

Reply:
I renamed the method, but since the unique uses came from the m-type cards (which we're deprecating), I'm afraid this rails helper won't be used.